nginx/modsecurity/ngx_http_modsecurity.c

```diff
@@ -1095,7 +1082,7 @@ ngx_http_modsecurity_create_ctx(ngx_http_request_t *r, ngx_time_t *start_time)
     cln->data = ctx;
 
     ctx->r = r;
-    ctx->start_time = start_time;
+    ctx->start_time = ngx_current_msec;
```

Comment:
This could lead to inaccuracies if we don't call `ngx_time_update()` ([linked here](https://github.com/nginx/nginx/blob/master/src/core/ngx_times.c#L81)), right?